package.json

```diff
@@ -31,7 +31,7 @@
   },
   "homepage": "https://github.com/bigcommerce/checkout-js#readme",
   "dependencies": {
-    "@bigcommerce/checkout-sdk": "^1.183.2",
+    "@bigcommerce/checkout-sdk": "^1.184.0",
```

Comment:
Can you please rebase this since SDK is already on `1.185.0` in master. https://github.com/bigcommerce/checkout-js/blob/master/package.json#L34